packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -98,9 +104,19 @@ export default {
     }
   },
 
+  mounted () {
+    this.isMobile = 'ontouchstart' in document.documentElement
+  },
+
   methods: {
```

Comment:
The methods isn't intuitive. The close won't really close, and what `isButton`? I believe it's here to detect whether users use keyboard to click the button, right?